### PR TITLE
Update behavior

### PR DESCRIPTION
**Pull request by pauladkisson**
This PR adds a trials table with remaining behavioral variables, and switches to pymatreader to accommodate both types of .mat files.

### CODE REVIEW

File: src/schneider_lab_to_nwb/schneider_2024/schneider_2024_convert_session.py [hunk 1]
```diff
@@ -12,14 +12,19 @@
 from schneider_lab_to_nwb.schneider_2024 import Schneider2024NWBConverter
 
 
-def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], stub_test: bool = False):
-
-    data_dir_path = Path(data_dir_path)
+def session_to_nwb(
+    recording_folder_path: Union[str, Path],
```
Comment (h-mayorquin) [inline src/schneider_lab_to_nwb/schneider_2024/schneider_2024_convert_session.py:16]: You can use str | Path instead of Union[str, Path] if you prefer btw.

File: src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml [hunk 2]
```diff
@@ -62,13 +84,6 @@ Behavior:
     - name: video_camera_2
       description: Two IR video cameras (AAK CA20 600TVL 2.8MM) are used to monitor the experiments from different angles of interest, allowing for offline analysis of body movements, pupillometry, and other behavioral data if necessary. Camera 2 is a zoomed-in view of the pupil of the mouse.
       unit: Frames
-  Devices:
```
Comment (h-mayorquin) [inline src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml:65]: Was this removal intended?

File: src/schneider_lab_to_nwb/schneider_2024/schneider_2024_notes.md [hunk 3]
```diff
@@ -1,9 +1,6 @@
 # Notes concerning the schneider_2024 conversion
 
 ## Behavior
-- opto events is just [0, 1] like empty variables (lick events), and opto_time in 'push' is all NaNs.  Where is the actual opto stim times recorded?
```
Comment (h-mayorquin) [inline src/schneider_lab_to_nwb/schneider_2024/schneider_2024_notes.md:4]: same here?

**Review by h-mayorquin — APPROVED** [review]
LGTM

Some minor comments.

Bigger commnent is unrelated to this PR though. After checking the state of the ndx-events 3.0 discussion I would advice to use ndx-events 2.0 which is already released as the former might still change quite a bit.

File: src/schneider_lab_to_nwb/schneider_2024/schneider_2024_convert_session.py (hunk 1 above)
Comment (pauladkisson) [reply src/schneider_lab_to_nwb/schneider_2024/schneider_2024_convert_session.py:16]: right! I'm still used to catering to python 3.9, but this conversion need not be so restricted.

File: src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml (hunk 2 above)
Comment (pauladkisson) [reply src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml:65]: Yes, you'll notice that these fields are moved up to line 55

File: src/schneider_lab_to_nwb/schneider_2024/schneider_2024_notes.md (hunk 3 above)
Comment (pauladkisson) [reply src/schneider_lab_to_nwb/schneider_2024/schneider_2024_notes.md:4]: Yes, these questions were resolved by new behavioral data + additional descriptions.

**Comment by pauladkisson** [comment]
> Bigger commnent is unrelated to this PR though. After checking the state of the ndx-events 3.0 discussion I would advice to use ndx-events 2.0 which is already released as the former might still change quite a bit.

Moving this to #11 for discussion.
